Gems/AtomLyIntegration/EMotionFXAtom/Code/Source/AtomActorDebugDraw.cpp

```diff
@@ -175,6 +185,13 @@ namespace AZ::Render
         }
     }
 
+    AzFramework::DebugDisplayRequests* AtomActorDebugDraw::GetDebugDisplay(AZ::s32 viewportId)
```

Comment:
nit: I think it's preferred to use the alias `AzFramework::ViewportId` (it's just an alias/typedef but might get promoted to a type in future and using it directly will make things easier in future).

Reply:
good call